test/unit/generator/content/summary.test.js

```diff
@@ -0,0 +1,14 @@
+const summary = require('../../../../app/generator/content/summary')
+const mockStatement = require('../../../mock-statement-data')
+
+describe('generate summary', () => {
+  test('includes header', () => {
```

Comment:
when `scheme.name` is `Sust...` and `scheme.year` is `2022`

could have another test where these are changed

Reply:
I did originally write a test for each known scheme but found as I was checking with a variable in the expect string they were effectively the same test over and over.

We could hard code the values into the test, but I don't want this service interface to be that tightly coupled to it's provider if it doesn't need to be.  eg if in later statement datasets, a decision is made to rename `Sustainable Farming Incentive` to `Farming Money Scheme`, as long as the contract is met in the message this would just map whatever is provided to the correct place.

I've renamed the test to make it a bit more descriptive.

Reply:
Ha realise I was talking about a totally different test...  Ignore 90% of what I said above.